supernode/server/api/api.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api
+
+var handlerCategories map[string][]*HandlerSpec
+
+const (
+	// VersionedPrefix is recommended, any new API should start with this prefix.
+	VersionedPrefix = "/api/v1"
```

Comment:
s/VersionedPrefix/V1VersionedPrefix
Maybe we will add v2,v3  API version in the future. 😄